packages/neuron-wallet/src/database/chain/subscriber/address-subscriber.ts

```diff
@@ -0,0 +1,10 @@
+import { EventSubscriber } from 'typeorm'
+import UserSettingSubscriber from './user-setting-subscriber'
+import AddressDescription from '../entities/address-description'
+
+@EventSubscriber()
+export default class AddressSubscribe extends UserSettingSubscriber<AddressDescription> {
+  listenTo() {
+    return AddressDescription
+  }
+}
```

Comment:
what does the subscriber is used for? It seems to do nothing but listens to changes from `AddressDescription`

Reply:
It is used to sync `address-description` from two db files. I will change the `AddressSubscribe` to `AddressDescriptionSubscribe`

Reply:
Got it, I found that the `UserSettingSubscriber` contains some reusable pieces.